src/db/tables/breaches.js

```diff
@@ -63,7 +63,25 @@ async function upsertBreaches (hibpBreaches) {
   })
 }
 
+/**
+ * Update logo path of a breach by name
+ *
+ * @param {string} name 
+ * @param {string} logoPath 
+ */
+async function updateBreachLogoPath(name, logoPath) {
+  await knex('breaches')
+    .where("name", name)
+    .update({
+      logo_path: logoPath,
+      // @ts-ignore knex.fn.now() results in it being set to a date,
+      // even if it's not typed as a JS date object:
+      modified_date: knex.fn.now()
```

Comment:
Blocking: I think I made a mistake in the typing here - we shouldn't be updating the `modified_date` ourselves, as it's a field from HIBP. Submitted https://github.com/mozilla/blurts-server/pull/3178 to fix the type error.

```suggestion
```